packages/plugin-infra/src/web3/useWeb3State.ts

```diff
@@ -1,7 +0,0 @@
-import { usePluginIDContext } from '.'
-import { useActivatedPluginWeb3State } from '..'
-
-export function useWeb3State(expectedPluginID?: string) {
```

Comment:
There's a vague meaning between `useWeb3State()` and `usePluginWeb3StateContext()`